DecipherNow/gm-fabric-dashboard
# 1123 fix: Ie11 svg elements focusable

Resolves #1123 

## Proposed Changes

 - gave special attribute `focusable: "false"` to svg icons.
 - gave special attribute `focusable: "false"` to sparkline svgs. 

 IE10, IE11, and Edge browsers will add SVG elements to the tab order by default unless you set focusable="false" to svg elements.  I have set all svg icons to `focusable:"false"` for now as icons come with aria-labels and should not be focusable unless they are links.  DocIcon on cards view are still focusable as they are wrapped in links.
  
  
## How to Test

  - open IE11
  - Fabric > JVM/GO > Routes View
  - check that only rows are selectable and svgs are no longer focusable
